Websites/bugs.webkit.org/PrettyPatch/PrettyPatch.rb

```diff
@@ -18,18 +18,6 @@ def self.prettify(string)
         string = normalize_line_ending(string)
         str = "#{HEADER}<body>\n"
 
-        # Just look at the first line to see if it is an SVN revision number as added
-        # by webkit-patch for git checkouts.
-        $svn_revision = 0
-        string.each_line do |line|
-            match = /^Subversion\ Revision: (\d*)$/.match(line)
-            unless match.nil?
-                str << "<span class='revision'>#{match[1]}</span>\n"
-                $svn_revision = match[1].to_i;
-            end
-            break
-        end
-
```

Comment:
I'm a little unclear what we're trying to do here, I think always comparing to `main` is a safe solution, though, and if there is a use case where we need comparison against a specific commit, we can deal with that later.